app/src/main/java/fr/free/nrw/commons/LocationPicker/LocationPickerActivity.java

```diff
@@ -104,6 +111,10 @@ public class LocationPickerActivity extends BaseActivity implements OnMapReadyCa
      * placeSelectedButton : fab for selecting location
      */
     FloatingActionButton placeSelectedButton;
+    /**
+     * cGPS: button for centre on location;
+     */
+    FloatingActionButton cGPS;
```

Comment:
Would you mind renaming the variable to `fabCenterOnLocation`? Unlike C++/etc, in Java the most common convention is to use long explicit names. Thanks!